src/Patch/Element.js

```diff
@@ -100,8 +100,14 @@ export default function(internals) {
           const content = isTemplate ? (/** @type {!HTMLTemplateElement} */
             (this)).content : this;
           /** @type {!Node} */
-          const rawElement = Native.Document_createElement.call(document,
-            this.localName);
+          let rawElement;
+          if (this.namespaceURI === document.namespaceURI) {
+            rawElement = Native.Document_createElement.call(document,
+              this.localName);
+          } else {
+            rawElement = Native.Document_createElementNS.call(document,
+              this.namespaceURI, this.localName);
```

Comment:
You can remove the if-else and use this line by itself: `createElementNS` can create elements in the HTML namespace too. (i.e. `document.createElement('button')` and `document.createElementNS('http://www.w3.org/1999/xhtml', 'button')` will both produce a regular HTML `<button>`.)